setup.cfg

```diff
@@ -1,3 +1,31 @@
+[metadata]
+classifiers =
+    Environment :: Console
+    License :: OSI Approved :: Apache License
+    Intended Audience :: Developers
+    Natural Language :: English
+    Operating System :: Android
+    Operating System :: Microsoft :: Windows :: Windows 10
+    Operating System :: POSIX:: Linux
```

Comment:
Can we say something like `Operating System :: POSIX:: *` ? We aim to support all POSIXen that ppl want to use neovim on (including at least the BSD:s and macOS).

Reply:
So unfortunately to show up in every subset of POSIX we'd have to explicitly list every single one. Alternatively, I'd imagine it would get the point across if the word Linux is simply removed.

Reply:
yes if `Operating System :: POSIX` works that should be enough.